docs/design/services/token-service/alias-configuration.md

```diff
@@ -1,67 +1,148 @@
 # Alias Configuration
 
 ## Purpose
-Account aliases allow an account to be referenced via a different id than the `<shard>.<realm>.<num>` accountId format which is native to the Hedera ledger.
-As part of [HIP 32](https://github.com/hashgraph/hedera-improvement-proposal/blob/master/HIP/hip-32.md) the functionality was made available via the auto-create flow in which an ED25519 or ECDSA public key was provided and used as the alias.
+Account aliases allow an account to be referenced via a different id than the `<shard>.<realm>.<num>` account ID format which is native to the Hedera ledger.
+As part of [HIP 32](https://github.com/hashgraph/hedera-improvement-proposal/blob/master/HIP/hip-32.md) the functionality was made available via the auto-create flow in which an `ED25519` or `ECDSA` public key was provided and used as the alias.
 
-To support greater scenario especially smart contract related scenarios for ECDSA based key the ledger needs to support additional ways of setting the alias at creation.
+To support greater scenarios, especially smart contract related scenarios for ECDSA-based key, the ledger needs to support additional ways of setting the alias at creation.
 
-This document explains how additional Crypto transaction flows may be used to create an account with an alias.
+This document explains how additional crypto and smart contract service transaction flows may be used to create an account with an alias.
+
+## Prerequisite reading
+* [HIP 32](https://github.com/hashgraph/hedera-improvement-proposal/blob/master/HIP/hip-32.md)
```

Comment:
Should probably refer directly to `hips.hedera.com` like the next line.  (Also, nit: `HIP-32` with dash for consistency.)

Reply:
Fixed.